{"name": "Try to get new pass", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: //h4[contains(text(), 'E-mail подтверждение для alltay@ya.ru отправлено.')]\n\n", "trace": "  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1456, in run\n    match.run(runner.context)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1903, in run\n    self.func(context, *args, **kwargs)\n  File \"step_test.py\", line 68, in step\n    \"//%s[contains(text(), '%s')]\" % (tag, text))\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 855, in find_element\n    'value': value})['value']\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 308, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 194, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given website 'https://www.mirkrestikom.ru/'", "status": "passed", "start": 1518611640466, "stop": 1518611650133}, {"name": "Then push link with text 'вход'", "status": "passed", "start": 1518611650133, "stop": 1518611651657}, {"name": "Then insert in 'email' 'alltay@ya.ru'", "status": "passed", "start": 1518611651657, "stop": 1518611651763}, {"name": "Then сlick on input button 'Выслать пароль'", "status": "passed", "start": 1518611651763, "stop": 1518611653489}, {"name": "Then find element 'h4' with text 'E-mail подтверждение для alltay@ya.ru отправлено.'", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: Unable to locate element: //h4[contains(text(), 'E-mail подтверждение для alltay@ya.ru отправлено.')]\n\n", "trace": "  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1456, in run\n    match.run(runner.context)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\behave\\model.py\", line 1903, in run\n    self.func(context, *args, **kwargs)\n  File \"step_test.py\", line 68, in step\n    \"//%s[contains(text(), '%s')]\" % (tag, text))\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 855, in find_element\n    'value': value})['value']\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 308, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 194, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1518611653490, "stop": 1518611653536}], "attachments": [{"name": "screenshot", "source": "ce984844-978c-4158-adca-85f8f8ac6eb3-attachment.png", "type": "image/png"}], "start": 1518611640466, "stop": 1518611653801, "uuid": "212c7bc7-26ec-4e3a-aa2c-722b693d6d44", "historyId": "943137b5b109847c21d55d60e1ec809b", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "Checking forget page (step)"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}